src/assets/i18n/en.json

```diff
@@ -3,7 +3,11 @@
     "movieList": {
       "title": "Movies",
       "labels": {
-        "itemQuantity": "items"
+        "itemQuantity": "items",
+        "page": "Page: {{page}} of {{totalPage}}",
+        "itemsPerPage": "Items per page:",
+        "lastPage": "Last page",
+        "nextPage": "Next page"
```

Comment:
ý a là mấy đống ni dùng chung cho toàn app nè, nên bỏ ở material chứ ko phải shows module